cli/src/commands/git/clone.rs

```diff
@@ -63,6 +64,9 @@ pub struct GitCloneArgs {
     /// Create a shallow clone of the given depth
     #[arg(long)]
     depth: Option<NonZeroU32>,
+    /// Run git clone as a forked git process
+    #[arg(long)]
+    shell: bool,
```

Comment:
suggestion: I'd recommend you actually drop the `--shell` argument altogether (from here and elsewhere). For one-off invocations, I believe you can write something like `jj --config='git.shell = true` git push`. I agree with your comment in the top-level thread that it seems unlikely to want to toggle it frequently, so using the more verbose `--config` version of the command seems like an acceptable trade-off vs polluting the command-line interface.